### PR TITLE
Bug 1873303: Protractor - added wait for Launcher Card to not initially exist, then is displayed when CR exists

### PR DESCRIPTION
**Pull request by dtaylor113**

### CODE REVIEW

File: frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts [hunk 1]
```diff
@@ -174,6 +174,7 @@ describe('Project Dashboard', () => {
     });
 
     it('is displayed when CR exists', async () => {
+      await browser.wait(until.not(until.presenceOf(projectDashboardView.launcherCard)));
```
Comment (spadgett) [inline frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts:177]: Were you able to reproduce? I'm not sure this is the issue. I suspect the `isPresent` call below happens asynchronously, and `createResource` completes too fast in some cases.

Comment (dtaylor113) [reply frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts]: ok, fixed to use `await`

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**